packages/components/Checkbox/SPEC.md

```diff
@@ -50,6 +50,68 @@ Win32:
 <Checkbox label="Circular Checkbox" shape="circular" />
 ```
 
+Android:
+
+##### Light Mode
+
+---
+
+<br/>
+
+![Checkbox with text on Android example](./assets/Checkbox_example_android.png)
+
+```jsx
```

Comment:
While I'm happy we're adding more assets to show what this looks like, I do feel like it's not scalable for the 5 platforms we support. However, we've been pretty slow on the "get a website with all the Fluent documentation train" up and running, so I guess it's worth taking anything that improves the current state of the world :)

Reply:
Correct , looks unscalable , we have come up some solution for screenshot images , generally if web platform renders there component and support editors like Codesandbox/codepen for editor there and see how the component looks like. 
But not sure if we can have that for mobile platforms, mostly i have seen screenshots for RN component libraries if they're not supported on Web otherwise, we would chose it and render component from spec.md documentation.